.github/workflows/pull-request.yml

```
@@ -0,0 +1,28 @@
+name: Pull Request
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  windows_build:
+    strategy:
+      matrix:
+        configuration: [Debug, Release]
+
+    runs-on: windows-2019
```

Did you want to run specifically on window-2019 and we have to manually upgrade to 2022? If we use `windows-latest`, they will move us over automatically.

Latest currently runs on `windows-2019`.

```suggestion
    runs-on: windows-latest
```

---

Yes, for this PR, we need to run on VS 2019. In the next PR, we will move to VS2022.